src/Alex/Gamestates/MainMenu/Options/OptionsStateBase.cs

```diff
@@ -118,8 +118,18 @@ protected override void OnInit(IGuiRenderer renderer)
 
                 if (canConstruct)
                 {
-                    state = (TGameState) constructor.Invoke(passedParameters);
-                    break;
+                    try
```

Comment:
I think we should just fix the error instead of catching it honestly, it's alright if it crashes here i feel. As long as it doesn't crash mid-game.